style.css

```diff
@@ -0,0 +1,100 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap');
+
+.wrapper {
+  position: absolute;
+  background: #fff;
+  width: 375px;
+  height: 718px;
+  background-image: url(images/header-background-mobile.svg);
+}
+
+header .nav-container {
+  position: absolute;
+  display: flex;
+  justify-content: space-between;
+  width: 375px;
+  height: 48px;
+  top: 44px;
+}
+
+.logo {
+  width: 100px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  padding: 10px 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  margin-left: 12px;
+}
+
+.hamburger {
+  width: 40px;
+  height: 40px;
+  padding: 8px;
+}
+
+.hamburger image {
+  width: 24px;
+  height: 24px;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 604px;
+  left: calc(50% - 375px / 2);
+  top: 92px;
+  background-image: url(images/header-shapes\ mobile@2x.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
- Great effort on your design! :fire: I noticed that there is some white space above your header and on the right side of your design, which differs from the given design. To make your design more aligned with the expected outcome, I suggest considering the following improvements: :smile: 

- [ ] Minimize the excess white space: Try reducing the amount of white space above the header and on the right side of the design. This will help create a more compact and visually balanced layout. :+1: 

- [ ] Optimize for mobile view: Ensure that your design fills the page appropriately when viewed on mobile devices. This will provide a consistent and seamless experience across different screen sizes. :100: 

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://github.com/anshuman7negi/my-portfolio/assets/102699958/c247e06d-0e42-456e-be30-c29dc57e951c) | ![image](https://github.com/anshuman7negi/my-portfolio/assets/102699958/41652bff-939c-44e5-bf4c-70ec713239fd) | 

- Keep up the good work! If you have any questions or need further assistance, I'm here to help. :wink: